cmd/krel/cmd/automate_beforefinal.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/release/cmd/krel/cmd/templates"
+	"k8s.io/release/pkg/automate"
+)
+
+var beforeFinalCmd = &cobra.Command{
+	Use:           "before-final",
+	Short:         "Arrange docs PR after code freeze",
```

Comment:
```suggestion
	Use:           "after-code-freeze",
	Short:         "Arrange docs PR after code freeze",
```
Does this make it easier to use, together with `after-enhancements-freeze`?

Reply:
I think it is clearer